src/platform/silabs/BLEManagerImpl.h

```diff
@@ -25,15 +25,15 @@
 #pragma once
 #if CHIP_DEVICE_CONFIG_ENABLE_CHIPOBLE
 
-#if (BLE_ENABLE || RSI_BLE_ENABLE)
+#if (SLI_SI91X_ENABLE_BLE || RSI_BLE_ENABLE)
 #define BLE_MIN_CONNECTION_INTERVAL_MS 45 // 45 msec
 #define BLE_MAX_CONNECTION_INTERVAL_MS 45 // 45 msec
 #define BLE_SLAVE_LATENCY_MS 0
 #define BLE_TIMEOUT_MS 400
 #endif // RSI_BLE_ENABLE
```

Comment:
comment needs to be update here and for the #endif associated with the #if of line 36

Reply:
updated comments.